tests/common.rs

```diff
@@ -105,15 +152,42 @@ macro_rules! tester {
     }};
 }
 
+#[cfg(feature = "rustls-tls")]
 #[macro_export]
 macro_rules! local_tester {
     ($f:ident, $endpoint:expr) => {{
-        let port: u16 = common::setup_server();
+        let port = common::setup_server();
+        let url = common::make_url($endpoint);
+        let caps = common::make_capabilities($endpoint);
         let f = move |c: Client| async move { $f(c, port).await };
-        tester!(f, $endpoint)
+        #[cfg(feature = "rustls-tls")]
+        tester_inner!(f, common::make_client(url, caps.clone(), "rustls"));
+        #[cfg(feature = "openssl-tls")]
+        tester_inner!(f, common::make_client(url, caps, "openssl"))
     }};
 }
 
+// #[cfg(feature = "openssl-tls")]
```

Comment:
Why did this go away?

Reply:
I don't know why I commented it out since we need just one local_tester, sorry about that. I fixed it now